### PR TITLE
Zip 1280 - Update scripts to support -f to go direct to log

### LINKED ISSUE #1280: zowe-copy-to-jes.sh creates a separate log
If you run zowe-install-proc.sh, or zowe-install-xmem.sh then it will log some stuff, then call zowe-copy-to-jes which creates a separate log file. This can make it tricky to stitch them together and debug each step.

### PR DESCRIPTION
**Pull request by stevenhorsman**
Signed-off-by: stevenhorsman <steven@uk.ibm.com>

#### PR type
- [X] Bugfix

#### Relevant issues
Fixes #1280 #801 

#### Changes proposed in this PR
- Update logging so that subscripts called append/embed into the main logging script

### CODE REVIEW

**Comment by jackjia-ibm** [comment]
@stevenhorsman I later realize the variable `zowe_logs_dir` defined in playbook could be misleading. The user may think it's all zowe logs, including runtime logs, but actually it's just install/confgure logs. So maybe `zowe_install_logs_dir` is more accurate?

**Comment by stevenhorsman** [comment]
Tested in convenience: https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-1300/3/ and ptf: https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-1300/4/

File: install/zowe-install.sh [hunk 1]
```diff
@@ -11,16 +11,17 @@
...
 fi
 
-while getopts "h:i:l:d" opt; do
+while getopts "f:h:i:l:d" opt; do
   case $opt in
     d) # enable debug mode
       # future use, accept parm to stabilize SMPE packaging
       #debug="-d"
       ;;
+    f) LOG_FILE=$OPTARG;;
```
Comment (jackjia-ibm) [inline install/zowe-install.sh:24]: I found not like other configuration scripts, `zowe-install-proc.sh` etc, this script is added a `-f` option. I know why we need `-f` for `zowe-copy-to-JES.sh`, but didn't quite get the purpose of here. But I can see it's not harmful at all.

Comment (stevenhorsman) [reply install/zowe-install.sh:24]: Hey Jack - this is to meet the requirement #801 that Onno had during the smpe packaging, so I've not doc'd it as I consider it internal use. Apologies for the lack of clarity, I've add a comment to explain

**Pull request merged**